pages/glossary/explicit-role.md

```diff
@@ -11,6 +11,6 @@ The [role attribute](https://www.w3.org/TR/role-attribute/) takes a list of toke
 
 Other roles may be added as they become available. Not all roles will be supported in all assistive technologies. Testers are encouraged to adjust which roles are allowed according to the [accessibility support base line](https://www.w3.org/TR/WCAG-EM/#step1c). For the purposes of executing test cases in all rules, it should be assumed that all roles are supported by assistive technologies so that none of the roles fail due to lack of accessibility support.
 
-#### Accessibility Support
+#### Accessibility Support for Definition of Explicit Role
```

Comment:
It would be better if you use small letter for word "Definition" and/or "Accessible".
Suggestion is to use one of the following: 
- Accessibility Support for defination of _accessible name_
- Accessibility Support for defination of Accessibile Name